testing/constraints-3.9.txt

```diff
@@ -21,7 +21,7 @@ pydata-google-auth==1.8.2
 requests==2.27.1
 scikit-learn==1.2.2
 sqlalchemy==1.4
-sqlglot==23.6.3
+sqlglot==26.1.3
```

Comment:
We try to keep this and our lower-bound in sync so that we have at least one test session that runs with our advertised lower-bound.

Please either:

1. Revert this change
2. Update the lower bound in `setup.py`.